manifests/a/Anker/AnkerMake/1.5.22/Anker.AnkerMake.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.7-4-3.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Anker.AnkerMake
+PackageVersion: 1.5.22
+InstallerLocale: en-US
+InstallerType: nullsoft
+InstallerSwitches:
+  Silent: /S
+UpgradeBehavior: install
+ReleaseDate: 2024-05-27
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/ankermake/AnkerMake-PrusaSlicer-Release/releases/download/v1.5.22/AnkerMake_Studio_V1.5.22.11_E1.7.1.exe
+  InstallerSha256: CD85ADCCCED54D13C40B22C0A51735C8A2EC827301BCEB7F237487C205684F73
```

```suggestion
Installers:
- Architecture: x64
  InstallerUrl: https://github.com/ankermake/AnkerMake-PrusaSlicer-Release/releases/download/v1.5.22/AnkerMake_Studio_V1.5.22.11_E1.7.1.exe
  InstallerSha256: CD85ADCCCED54D13C40B22C0A51735C8A2EC827301BCEB7F237487C205684F73
  AppsAndFeaturesEntries:
  - DisplayVersion: V_1.5.22.11_E.7.1
    Publisher: My company, Inc.
    ProductCode: AnkerMake Studio
  ProductCode: AnkerMake Studio
```
AppsAndFeaturesEntries are necessary as the version written to the registry does not match the `PackageVersion`